commcare_connect/opportunity/visit_import.py

```diff
@@ -192,12 +195,26 @@ def _bulk_update_payments(opportunity: Opportunity, imported_data: Dataset) -> P
     seen_users = set()
     missing_users = set()
     with transaction.atomic():
+        messages = []
         usernames = list(payments)
         users = OpportunityAccess.objects.filter(user__username__in=usernames, opportunity=opportunity).select_related(
             "user"
         )
         for access in users:
-            Payment.objects.create(opportunity_access=access, amount=payments[access.user.username])
-            seen_users.add(access.user.username)
+            username = access.user.username
+            amount = payments[username]
+            Payment.objects.create(opportunity_access=access, amount=amount)
+            seen_users.add(username)
+            messages.append(
+                Message(
+                    usernames=[username],
+                    title=gettext("Payment received"),
+                    body=gettext(
+                        f"You have received a payment of {access.opportunity.currency} {amount} for "
+                        f"{access.opportunity.name}. Click on this notification for more information on the payment."
```

Comment:
Just thinking about the language here. I think it would be clear to say "Payment Sent" since they won't have received anything yet.

"A payment request of ... has been created for ......"

Reply:
This notification is sent after the payment data is uploaded. So the users would have already received the payment by the time they receive this notification.